solarforecastarbiter/io/api.py

```diff
@@ -927,10 +927,23 @@ def post_raw_report_processed_data(self, report_id, raw_report):
             obs_post = self.post(
                 f'/reports/{report_id}/values',
                 json=obs_data, headers={'Content-Type': 'application/json'})
+            if fxobs.original.reference_forecast is not None:
+                ref_fx_data = {
+                    'object_id': fxobs.original.reference_forecast.forecast_id,
+                    'processed_values': serialize_timeseries(
+                        fxobs.reference_forecast_values)}
+                ref_fx_post = self.post(
+                    f'/reports/{report_id}/values',
+                    json=ref_fx_data)
```

Comment:
the `post` calls above specify `headers={'Content-Type': 'application/json'}` so I expected that here. If not needed, add comment on why.